app/models/validators/aaib_report_validator.rb

```diff
@@ -9,5 +9,5 @@ class AaibReportValidator < SimpleDelegator
   validates :summary, presence: true
   validates :body, presence: true, safe_html: true
 
-  validates :date_of_occurrence, presence: true, date: true
+  validates :date_of_occurrence, date: true, allow_blank: true
```

Comment:
I agree, this could lead to a user error when adding a standard report which must have a date of occurrence.